src/Admin/assets/css/admin-styles.css

```diff
@@ -82,56 +132,34 @@
     margin: 0;
 }
 
-/* Responsive Design */
-@media (max-width: 768px) {
-    .metric-box {
-        flex: 1 1 calc(50% - 15px); /* Two boxes per row on smaller screens */
-    }
-}
-
-@media (max-width: 480px) {
-    .metric-box {
-        flex: 1 1 100%; /* One box per row on very small screens */
-    }
-}
-
-/* Scan Scheduling Section */
-.scan-scheduling-section {
-    background: #fff;
-    border: 1px solid #e5e5e5;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-    padding: 10px;
-    margin-bottom: 20px;
+.chart-box {
+    grid-row: span 2;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
 }
 
-.scan-scheduling-section .heading-box {
-    background-color: #f9f9f9;
-    color: #fff;
-    padding: 15px;
+canvas {
```

Comment:
if possible add class or id instead of using flat tag

Reply:
@rajanvijayan You mean to remove all "!important" from css

Reply:
> if possible add class or id instead of using flat tag

Using flat tags for the table and its elements makes it unnecessary to add classes to every tag